study.md

```diff
@@ -50,7 +50,24 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+{
+  "kitchen": {
+    "cupboard": {
+      "bowls": "4",
+      "plates": "100"
+    }
+    "electric stove": {
+      "heat1": "small",
+      "heat2": "medium",
+      "heat3": "large",
+      "heat4": "extra large"
+    }
+    "spice rack":{
+      "rack1": "chives, lemongrass, mint",
```

Comment:
You would want these items to be in an array so that you can access them one at a time. As it is now, they are all in one string.